tests/queries/get-return-type.test.ts

```diff
@@ -0,0 +1,62 @@
+import testObject from '../../src/queries';
+import { transformToAst } from '../helpers';
+
+const testFunctionName = "getReturnType";
+
+describe(testFunctionName, () => {
+    test(`should return information no return type annotation`, () => {
```

Comment:
maybe you should use singlequote not a template string in all tests?

Reply:
Tests are out of eslint range and standards.

Reply:
👍